examples/nlp/language_modeling/conf/megatron_gpt_inference.yaml

```diff
@@ -3,7 +3,7 @@ inference:
   top_k: 0  # The number of highest probability vocabulary tokens to keep for top-k-filtering.
   top_p: 0.9 # If set to float < 1, only the most probable tokens with probabilities that add up to top_p or higher are kept for generation.
   temperature: 1.0 # sampling temperature
-  add_BOS: True # add the bos token at the begining of the prompt
+  add_BOS: False # add the bos token at the begining of the prompt
```

Comment:
do we need to change this?

Reply:
Yes, for example all our recent models will do really poorly with `add_BOS=True` because the model never sees BOS. This just happens to be fine for the GPT2 tokenizer because bos = eos = |<endoftext>|.